instance/tasks.py

```diff
@@ -27,7 +27,7 @@
 from django.conf import settings
 
 from instance.github import get_username_list_from_team, get_pr_list_from_username
-from instance.models.instance import SingleVMOpenEdXInstance
+from instance.models.openedx_instance import SandboxInstance
```

Comment:
I seem to recall it's difficult to put models in a sub-module of `models` in Django, but I could be mistaken.

What I'd really like to do is make `instance` very generic and move all the openedx-related code into a separate app like `openedx_instance.models` or `edxinstance.models` or something. But I wasn't sure I'd have scope to fully do that change along with all the other changes I already took on for this PR.